apriltag.h

```diff
@@ -130,39 +130,57 @@ struct apriltag_detector
     // User-configurable parameters.
 
     // How many threads should be used?
+    //
+    // Default: 1
     int nthreads;
 
     // detection of quads can be done on a lower-resolution image,
     // improving speed at a cost of pose accuracy and a slight
     // decrease in detection rate. Decoding the binary payload is
-    // still done at full resolution. .
+    // still done at full resolution. Decimation of 2.0 will reduce
+    // image dimensions by half. Values <=1.0 are ignored.
+    // Valid decimation rates are 1.5 and positive integers
+    //
+    // Default: 2.0
     float quad_decimate;
 
     // What Gaussian blur should be applied to the segmented image
     // (used for quad detection?)  Parameter is the standard deviation
-    // in pixels.  Very noisy images benefit from non-zero values
-    // (e.g. 0.8).
+    // in pixels.  Very noisy images benefit from non-zero values (e.g. 0.8).
+    // Gaussian blur kernel size is (int)(4*fabssigma)). Add 1 if even.
+    // Negaive values will sharpen the image instead of blur
+    // Sample kernel sizes are below:
+    //
+    // max sigma          kernel size
+    // 0.0 -> 0.499       1  (disabled)
+    // 0.5 -> 0.999       3
+    // 1.0 -> 1.499       5
+    // 1.5 -> 1.999       7
```

Comment:
This table is a bit confusing. You are referring to the look-up table for equation "kernel_size = 4 * abs(quad_sigma)":
https://github.com/AprilRobotics/apriltag/blob/bee5efb1cc263ec1459bca46193e3428220d52ba/apriltag.c#L1020-L1027

But why this mapping from 0 to 0.499, 0.5 to 0.999, etc.? That looks like 0.0 gives you 0.499 and 0.5 gives you 0.999. Do you mean that all values in this range map to the same kernel size?

The notation is a bit confusing and I would personally prefer if this is just properly formalised. With an ASCII or LaTeX function.